www/partials/about_contact.html

```diff
@@ -24,7 +24,7 @@
          <li class="list-group-item"><a href="http://support.omniwallet.org" target="_new">Knowledge Base/Support Center</a>: This is a support knowledge base where you can review common topics, post questions asking for help or help answer other users questions</li>
          <li class="list-group-item">Email: You can open a support ticket by emailing us at <img src="/assets/img/contact.png" width="150" height="28"> </li>
 	 <li class="list-group-item"> <a href="http://www.hipchat.com/gIJc11cbH" target="_new">Live Chat</a>: Chat with the developers, share your ideas or ask questions.<br />
-                                     <small>Availability varies, but we should generally be around <b>Mon-Fri 9-5 CDT</b></small></li>
+                                     <small>Availability varies, but we should generally be around <b>Mon-Fri 9-5 EST</b></small></li>
```

Comment:
Maybe it's useful for international users, if the time were given in UTC as well.